07/reshape-a-matrix.checkpoint.md

```diff
@@ -144,67 +152,79 @@ def reshape_matrix(matrix, r, c):
 ```
 
 ##### !end-placeholder
-
 ##### !tests
 
 ```py
 import unittest
 from main import reshape_matrix
 class TestPython1(unittest.TestCase):
-    def test_example_one(self):
+    def test_1_by_4_can_reshape_into_4_by_1(self):
         # Arrange
-        matrix = [[1,2],[3,4]]
-        r = 1
-        c = 4
+        matrix = [[1], [2], [3], [4]]
+        rows = 1
+        columns = 4
+
         # Act
-        answer = reshape_matrix(matrix, r, c)
+        result = reshape_matrix(matrix, rows, columns)
+
         # Assert
-        self.assertEqual([[1,2,3,4]], answer)
+        assert result == [[1, 2, 3, 4]]
```

Comment:
Is it safe to switch from the actual `unittest` checkers to a plain `assert` for the tests that Learn actually runs?